start.js

```diff
@@ -59,8 +59,16 @@ function stop (message) {
 
 async function runFastify (args) {
   require('dotenv').config()
-
   const opts = parseArgs(args)
+
+  if (opts.beforeModule) {
```

Comment:
how would this be handled for the `watch` option?

Reply:
Watch option checks for changes in `process.cwd()` so currently for both this addition and the custom logger module it won't be triggered if it's outside of the current working directory. Should both of these files be watched explicitly as well?